prow/cmd/deck/static/prow/prow.ts

```diff
@@ -648,20 +667,34 @@ function redraw(fz: FuzzySearch): void {
         max = 2 * 3600;
     }
     drawJobHistogram(totalJob, jobHistogram, now - (12 * 3600), now, max);
+    if (rerunStatus != null) {
+        modal.style.display = "block";
+        rerunCommand.innerHTML = `Nice try! The direct rerun feature hasn't been implemented yet, so that button does nothing.`;
+    }
+
 }
 
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
-    const url = `https://${window.location.hostname}/rerun?prowjob=${prowjob}`;
+    const url = `http://${window.location.hostname}/rerun?prowjob=${prowjob}`;
     const c = document.createElement("td");
     const i = icon.create("refresh", "Show instructions for rerunning this job");
+    const login = getCookieByName("github_login");
     i.onclick = () => {
         modal.style.display = "block";
         rerunElement.innerHTML = `kubectl create -f "<a href="${url}">${url}</a>"`;
         const copyButton = document.createElement('a');
         copyButton.className = "mdl-button mdl-js-button mdl-button--icon";
-        copyButton.onclick = () => copyToClipboardWithToast(`kubectl create -f "${url}"`);
+        copyButton.onclick = () => copyToClipboardWithToast(`kubectl create -f "${url}" aldaslkfjalskfj`);
         copyButton.innerHTML = "<i class='material-icons state triggered' style='color: gray'>file_copy</i>";
         rerunElement.appendChild(copyButton);
+        const runButton = document.createElement('a');
+        runButton.innerHTML = "<button class='mdl-button mdl-js-button'>Run</button>";
+        if (login === "") {
+            i.href = `https://${window.location.hostname}/github-login?rerun_status=work_in_progress`;
+        } else {
+            runButton.href = `https://${window.location.hostname}/?rerun_status=work_in_progress`;
```

Comment:
On further thought, I'm also not a huge fan of this being a link at all, but perhaps that's going to change?

Reply:
In the future, it will link to the /rerun endpoint and rerun the job, as in Alvaro's PR

Reply:
Did we decide against a dialog in the page? I haven't checked the design doc recently.

Reply:
We will have a dialog: 
![image](https://user-images.githubusercontent.com/28876252/59634945-c30fcc80-9104-11e9-9bfc-8cff4b60fcce.png)

runButton will become the button in the bottom right. After the test is triggered, we'll have another dialog saying it was successful.